fuel-indexer-macros/src/indexer.rs

```diff
@@ -101,6 +102,41 @@ fn decode_snippet(ty_id: u64, ty: &Ident, name: &Ident) -> proc_macro2::TokenStr
     }
 }
 
+fn process_block_fn_items(input: &ItemFn) -> TokenStream2 {
```

Comment:
I could've hacked this into `process_fn_items`, but wanted to be explicit and keep this separate